src/controller/CHIPDeviceController.cpp

```diff
@@ -947,7 +947,8 @@ DeviceCommissioner::ContinueCommissioningAfterDeviceAttestation(DeviceProxy * de
         return CHIP_ERROR_INCORRECT_STATE;
     }
 
-    if (mCommissioningStage != CommissioningStage::kAttestationVerification)
+    if (mCommissioningStage != CommissioningStage::kAttestationVerification &&
+        mCommissioningStage != CommissioningStage::kAttestationRevocationCheck)
```

Comment:
Again: How would we get here when `mCommissioningStage == CommissioningStage::kAttestationVerification`?  We don't call into the delegate in that stage anymore.

Reply:
Fixed in latest commit.